app/config/settings.py

```diff
@@ -1041,6 +1042,14 @@ def strtobool(val) -> bool:
     *blocklist,
 }
 
+# GitHub App
+GITHUB_APP_INSTALL_URL = os.environ.get("GITHUB_APP_INSTALL_URL", "")
+GITHUB_APP_ID = os.environ.get("GITHUB_APP_ID", "")
+GITHUB_CLIENT_ID = os.environ.get("GITHUB_CLIENT_ID", "")
+GITHUB_CLIENT_SECRET = os.environ.get("GITHUB_CLIENT_SECRET", "")
+GITHUB_PRIVATE_KEY_PATH = os.environ.get("GITHUB_PRIVATE_KEY_PATH", "")
```

Comment:
Rather than using a path could we include the key as an environment variable? Easier to deploy without data volumes then.

Reply:
I'm not sure how well that would work with the length of the key and the multiline nature?

Reply:
It's possible (use single quotes in your `.env` file), but base64 encoding might be more robust if you want to go down that route.

Reply:
Did you try doing this?

Reply:
Ah sorry, I missed that comment. I'll try it now.